pkg/ghcache/ghcache.go

```diff
@@ -27,7 +27,7 @@ type blob struct {
 	Issue               github.Issue
 }
 
-func PullRequestsGet(ctx context.Context, dv *diskv.Diskv, c *github.Client, t time.Time, org string, project string, num int) (*github.PullRequest, error) {
+func PullRequestsGet(ctx context.Context, dv *diskv.Diskv, c *github.Client, t time.Time, org, project string, num int) (*github.PullRequest, error) {
```

Comment:
I'll be honest - I don't actually agree with this style change. I understand it's possible in Go, but I don't feel like this change makes the function signature any more readable. I think we should filter out this particular golangci-lint suggestion - do you know which checker it was?

Reply:
I will revert that and add the nolint rule for that

Reply:
done.
thanks for your review again :)